manifests/c/CedricLouvrier/Pingo/1.23.9/CedricLouvrier.Pingo.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: CedricLouvrier.Pingo
+PackageVersion: 1.23.9
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: pingo.exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://css-ig.ye4241.workers.dev/bin/pingo-win64.zip

```

Comment:
```suggestion
  InstallerUrl: https://css-ig.net/bin/pingo-win64.zip
```

Reply:
![CleanShot 2024-09-07 at 21 40 33@2x](https://github.com/user-attachments/assets/b4823362-4e38-481d-a46d-342412f92c0b)

This link needs JavaScript to download real zip file, therefore, I use CloudFlare to resolve this issue.